### PR TITLE
Added settings for tile indicator fill colors

### PR DESCRIPTION
**Pull request by JoshJ96**
Commit based on this issue: https://github.com/runelite/runelite/issues/14445

My first commit :) happy to work with you guys

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsOverlay.java [hunk 1]
```diff
@@ -106,6 +106,9 @@ private void renderTile(final Graphics2D graphics, final LocalPoint dest, final
 			return;
 		}
 
+		graphics.setColor(new Color(fillColor.getRed(), fillColor.getGreen(), fillColor.getBlue(), fillColor.polyAlpha()));
+		graphics.fillPolygon(poly);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsOverlay.java:110]: `renderPolygon` is already doing a `fill` with the polygon with `new Color(0, 0, 0, 50)` - you should not be drawing over it again but instead call the overloaded `renderPolygon` which accepts a fill color.

File: runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java [hunk 2]
```diff
@@ -55,12 +55,22 @@ default boolean highlightDestinationTile()
 	{
 		return true;
 	}
+	@ConfigItem(
+		keyName = "destinationTileFillColor",
+		name = "Destination tile fill color",
+		description = "Configures the fill color of destination tile",
+		position = 3
+	)
+	default boolean destinationTileFillColor()
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java:64]: all of these should probably have `@Alpha`

Comment (JoshJ96) [reply runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsConfig.java]: Sounds good ill update and do another commit with that

**Comment by JoshJ96** [comment]
All done with commits for tonight. Let me know what you think!

**Comment by Adam-** [comment]
The default behavior should be the existing behavior, which is the fill color is `0,0,0,50`

**Comment by JoshJ96** [comment]
Updated

**Comment by JoshJ96** [comment]
> The default behavior should be the existing behavior, which is the fill color is `0,0,0,50`



> The default behavior should be the existing behavior, which is the fill color is `0,0,0,50`

Updated

**Pull request merged**